.gitignore

```diff
@@ -7,7 +7,16 @@ local.properties
 .idea/
 *.iml
 
+# Eclipse
```

Comment:
Not sure what Studio-based should imply here... Is this not a usual Gradle project that can be handled by any Gradle-compatible tool incl. Eclipse?